product.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Vacay Store / Products</title>
+    <link href="https://vacay-ui.netlify.app/components/vacay.css" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header box-shadow">
+        <h1 class="logo">vacay</h1>
+        <div class="search-box">
+            <i class="fas fa-search"></i>
+            <input type="text" placeholder="Search">
+        </div>
+        <nav class="navigation">
+            <ul>
+                <li><a href="#" class="btn btn-primary">Login</a></li>
+                <li class="badge-wrapper">
+                    <a href="#"><i class="btn-icon far fa-heart"></i></a>
+                    <span class="badge badge-number">5</span>
+                </li>
+                <li>
+                    <a href="#" class="btn btn-secondary-icon-text-no-border">
+                        <div class="badge-wrapper">
+                            <i class="btn-icon fas fa-shopping-cart"></i>
+                            <span class="badge badge-number">5</span>
+                        </div>
+                        Cart
+                    </a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <div class="product-page-container">
+    <aside class="filter">
+            <div class="filter-header">
+                <h3>Filters</h3>
+                <a href="#" class="btn btn-primary-link">Clear</a>
+            </div>
+            <ul>
+                <h4>Price</h4>
+                <li>
+                    <input type="range" name="" id="">
+                </li>
+            </ul>
+            <ul>
+                <h4>Category</h4>
+                <li class="input input-checkbox-radio">
+                    <input type="checkbox">
+                    <label for="">Camera</label>
+                </li>
+                <li class="input input-checkbox-radio">
+                    <input type="checkbox">
+                    <label for="">Suitcase</label>
+                </li>
+            </ul>
+            <ul>
+                <h4>Rating</h4>
+                <li class="input input-checkbox-radio">
+                    <input type="radio">
+                    <label for="">4 Stars & above</label>
```

Comment:
All the radio buttons are getting clicked. Add the same value for the `name` attribute to all radio inputs.

Reply:
@ApurvaSawant11 can you check now, I have done the changes

Reply:
Yes it's working now